react_app/src/App.js

```diff
@@ -1,27 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import EmbeddingForm from "./Form";
+import "./App.scss";
 
-const App = () => {
+const DATASET_CREATE_STRING = "Create a dataset";
```

Comment:
Can you move these constants to their own file?

Reply:
Just removed instead - it's not used in the current build anyway